cmd/aws/lambda/autoscale/main.go

```diff
@@ -63,6 +63,10 @@ func init() {
 			panic(err)
 		}
 
+		if dps > math.MaxInt32 || dps < math.MinInt32 {
+			panic(fmt.Errorf("value %d out of bounds for int32", dps))
```

Comment:
Since this will crash the program, it would be helpful to add a prefix to the error to point out where the failure is -- maybe `init downscale:` would work? The alternative is to not crash and leave the default value in place, similar to this recommendation from CodeQL:
```go
const DefaultAllocate int32 = 256

func parseAllocateGood2(desired string) int32 {
	// GOOD: parse specifying the bit size
	parsed, err := strconv.ParseInt(desired, 10, 32)
	if err != nil {
		return DefaultAllocate
	}
	return int32(parsed)
}
```

Also applies to the other change. 👍

Reply:
Definitely makes sense - what do you think of handling such as that if provided value is < 1 (for both upscale and downscale) we just set to 1, and if above 30 (for upscale) or above 60 (for downscale) we set to the max value for each? I'm not sure if I love the idea - I don't have the full context on potential impact (especially for upscale at max value). Either way I do like the idea on not panicking and using a known good value, though.

Reply:
Yeah, something like that would work. These values can never be below 1, but their max value can be anything (although realistically, it would never be configured to be more than 60 or 120 minutes). 

For context, these values are used by CloudWatch alarms for both the evaluation period and the number of datapoints ([here's a reference](https://github.com/brexhq/substation/blob/main/cmd/aws/lambda/autoscale/main.go#L431-L432)) -- ex. the Kinesis stream utilization has to be below the threshold for (by default) 60 minutes with no interruptions to start a scale down event.

I'm a fan of crashing programs as an overall strategy, but in this case we have some reasonable defaults so I can see the argument for ignoring invalid input, logging an error, and using the default values. From an end-user point of view it's likely more important that the stream scale using _any_ strategy versus never scaling (which is what would happen if it were to crash). For logging, you can see an [example of that here](https://github.com/brexhq/substation/blob/main/cmd/aws/lambda/autoscale/main.go#L119-L120) (with more throughout the codebase). I recommend an info log entry for this case because they're always logged.